### DISCUSSION TITLE
copilot.vim sometimes insert arbitrary symbols

### DISCUSSION THREAD

**Discussion by ds82** [community/community#46966] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Sometimes copilot show the correct suggestion in neovim, like this:

![Screenshot 2023-02-10 at 13 05 55](https://user-images.githubusercontent.com/332905/218088598-46f4480e-245e-4f88-81ad-40867cf7e9c3.png)

But upon accepting this suggestion it inserts some odd symbols:

![Screenshot 2023-02-10 at 13 05 59](https://user-images.githubusercontent.com/332905/218088759-5e65bacc-14a5-4308-8124-96c7398f963a.png)

Any ideas why?

**Comment by ava5627** [comment]
check out [this discussion](https://github.com/community/community/discussions/29817) from a couple months ago

**Reply by tpope** [reply]
#29817 is correct. Specifically, this can happen when the `<Tab>` map is created with Neovim's APIs without `replace_keycodes = false` (which has traditionally been the default).

**Comment by rphuber** [comment]
this actually seems more similar to this issue #46277
